test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,181 @@
+require "test_helper"
+
+describe WorksController do
+
+  describe "index" do
+    it "should get index" do
+      # TODO: does it get index if there are 0 works?
+      get "/works"
+      must_respond_with :success
+    end
+  end
+
+  describe "show" do
+    before do
+      @work = Work.create(
+        category: "movie", 
+        title: "hello world", 
+        creator: "Yoyo", 
+        year: "2020", 
+        description: "Very nice"
+      )
+    end
+
+    it "will get show for valid ids" do
+      # Arrange
+      valid_work_id = @work.id
+      # Act
+      get "/works/#{valid_work_id}"
+      # Assert
+      must_respond_with :success
+    end
+
+    it "will respond with not_found for invalid ids" do
+      # Arrange
+      invalid_work_id = -1
+      # Act
+      get "/works/#{invalid_work_id}"
+      # Assert
+      must_respond_with :not_found
+    end
+
+  end
+
+  describe "new" do
+    it "can get the new_work_path" do
+      get new_work_path
+
+      must_respond_with :success
+    end
```

Comment:
This test is failing because you're requiring the user to log in first before they can get to the form.

This would also require you provide a user fixture. 

```suggestion
    it "can get the new_work_path" do
      perform_login()
      get new_work_path

      must_respond_with :success
    end
```